.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  tests:
+
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ubuntu-18.04
```

Reply:
Since we had an issue with tests on `ubuntu-latest` with MeiliSearch, I would rather we do tests on known versions. We need to change this in all the integration repositories as well, I haven't taken the time to do it yet 😅